highest_number_cubed/main.py

```diff
@@ -1,5 +1,14 @@
 def highest_number_cubed(limit):
-    pass
+    x = 0
+    while (x**3) < limit:
+        x += 1
+    return (x - 1)
+
```

Comment:
test_values = [] # will hold all test values
    test_value = 0 # initalize our starting value
    cubed_number = test_value 

    def cube_this(value):
        import math               # function that returns a cubed value
        return math.pow(value, 3)

    while cubed_number <= limit:
        test_values.append(test_value)
        test_value += 1
        cubed_number = cube_this(test_value)

    return max(test_values) # will return the highest value in the array

Here was my approach, yours was much simpler. Leave comments to help other developers understand your approach as well.